CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ and this project adheres to
 
 ## [unreleased]
 
+### Added
+
+- ​Added `opam-repositories` input to support multiple opam repositories.
+
+### Removed
+
+- The `opam-repository` input has been removed in order to add the
+  `opam-repository` input.
+- The cache steps for the opam download cache have been removed.
```

Comment:
Is this necessary?

Reply:
For example, just adding a multicore one somehow made it 300 MB: https://github.com/ocaml/setup-ocaml/runs/2772269027?check_suite_focus=true#step:19:4

Reply:
I think it's better for the user to be able to choose to switch it off, rather than it not be available?

Reply:
Now that all repositories will be purged every time, there is no chance that the cache will mess things up anymore, so we may restore the opam cache step. The cache size is still a problem, but I'm beginning to doubt that there are different reasons for the size growth.